sandbox/src/mocks/retrieveServiceRequests/responses/CancelledServiceRequest.json

```diff
@@ -0,0 +1,223 @@
+{
+    "resourceType": "Bundle",
+    "type": "searchset",
+    "timestamp": "2022-03-23T08:14:07.422+00:00",
+    "meta": {
+      "profile": ["https://fhir.hl7.org.uk/StructureDefinition/UKCore-Bundle"],
+      "lastUpdated": "2022-03-23T08:14:07.422+00:00"
+    },
+    "entry": [
+      {
+        "fullUrl": "_baseUrl_/ServiceRequest/r.6c2a1f9b-2517-419d-9099-591ba63f176c",
+        "search": {
+          "mode": "match"
+        },
+        "resource": {
+          "resourceType": "ServiceRequest",
+          "id": "r.6c2a1f9b-2517-419d-9099-591ba63f176c",
+          "meta": {
+            "versionId": "1",
+            "profile": ["https://fhir.nhs.uk/StructureDefinition/NHSDigital-ServiceRequest-PatientCare"]
+          },
+          "extension": [
+            {
+              "url": "https://fhir.nhs.uk/StructureDefinition/Extension-Portal-Link",
```

Comment:
I don't think a cancelled request would have the portal link as you can't see cancelled referrals in MYR.